x-pack/plugin/transform/qa/multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/transform/integration/TransformIT.java

```diff
@@ -83,76 +83,75 @@ public void cleanTransforms() throws Exception {
     }
 
     public void testTransformCrud() throws Exception {
-        String indexName = "basic-crud-reviews";
-        String transformId = "transform-crud";
-        createReviewsIndex(indexName, 100, NUM_USERS, TransformIT::getUserIdForRow, TransformIT::getDateStringForRow);
-
-        Map<String, SingleGroupSource> groups = new HashMap<>();
-        groups.put("by-day", createDateHistogramGroupSourceWithCalendarInterval("timestamp", DateHistogramInterval.DAY, null));
-        groups.put("by-user", new TermsGroupSource("user_id", null, false));
-        groups.put("by-business", new TermsGroupSource("business_id", null, false));
-
-        AggregatorFactories.Builder aggs = AggregatorFactories.builder()
-            .addAggregator(AggregationBuilders.avg("review_score").field("stars"))
-            .addAggregator(AggregationBuilders.max("timestamp").field("timestamp"));
-
-        TransformConfig config = createTransformConfigBuilder(
-            transformId,
-            "reviews-by-user-business-day",
-            QueryConfig.matchAll(),
-            indexName
-        ).setPivotConfig(createPivotConfig(groups, aggs)).build();
-
-        putTransform(transformId, Strings.toString(config), RequestOptions.DEFAULT);
-        startTransform(config.getId(), RequestOptions.DEFAULT);
-
-        waitUntilCheckpoint(config.getId(), 1L);
-
-        stopTransform(config.getId());
-        assertBusy(() -> { assertEquals("stopped", getTransformStats(config.getId()).get("state")); });
+        var configId = createStoppedTransform("basic-crud-reviews", "transform-crud");
```

Comment:
`configId` is just the same as `transformId`.
So if you have `transformId` set to `"transform-crud"` then you don't need to return the configId from the `createStoppedTransform` method.